homeassistant/components/notify/__init__.py

```diff
@@ -91,15 +91,13 @@ def setup(hass, config):
 
         def notify_message(notify_service, call):
             """Handle sending notification message service calls."""
-            message = call.data[ATTR_MESSAGE]
+            message = call.data.get(ATTR_MESSAGE)
+            title = call.data.get(ATTR_TITLE, ATTR_TITLE_DEFAULT)
 
-            title = template.render(
```

Comment:
Let's keep this logic in it. It has been there from before the template way got introduced. Would be too breaking change doing that now.